opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperTestDialog.tsx

```diff
@@ -113,13 +133,26 @@ const CsvMapperTestDialog: FunctionComponent<CsvMapperTestDialogProps> = ({
           </Button>
           {loading && (
             <div style={{ marginLeft: 10 }}>
-              <Loader variant={LoaderVariant.inElement} />
+              <Loader variant={LoaderVariant.inElement}/>
             </div>
           )}
         </div>
+        {result
+          && <div style={{
```

Comment:
Use classes instead of inline style. And check if you can use a MUI component instead of redefining something like that ?

Reply:
Ok sure. All the component have inline-style so I try to keep the styling consistent. So I will refactor and use MUI component and class.

Reply:
I use Box from MUI in order to style the div container, is it okay or you prefer to a Styled-component or MakeStyle instead ?